test1.js

```diff
@@ -3,3 +3,10 @@
 if(foo  === "bar") {
   console.log( 'hey' )
 }
+
+
+var foobar = '' ;
+
+if(foobar      === "bar") {
```

Comment:
Codacy found an issue: [Multiple spaces found before '==='.](https://app.dev.codacy.org/gh/machadoit-test-org/eslint-example/pullRequest?prid=26122)
```suggestion
if(foobar === "bar") {
```